lib/craco-less.js

```diff
@@ -18,10 +22,97 @@ const overrideWebpackConfig = ({ context, webpackConfig, pluginOptions }) => {
       githubIssueQuery,
     });
 
-  const lessExtension = /\.less$/;
-
   pluginOptions = pluginOptions || {};
 
+  const createLessRule = ({ baseRule, overrideRule }) => {
+    baseRule = clone(baseRule);
```

Comment:
I think the usage of `clone` here is unnecessary and is adding a dependency to the end users without a legit reason. Can you remove this line and add back `clone` to development dependencies since we are already using it in tests